app/templates/advice.html.twig

```diff
@@ -1,5 +1,6 @@
-<a id="advise" class="btn" href="javascript:void(0);">Advise me !</a>
-<a id="worst-advise" class="btn btn-cancel" href="javascript:void(0);">Prevent me to make him win !</a>
+<a id="advise" class="btn" href="javascript:void(0);">Make me Win</a>
+<a id="advise-opponent-lose" class="btn" href="javascript:void(0);">Make him lose</a>
+<a id="worst-advise" class="btn btn-cancel" href="javascript:void(0);">Stop me to make him win</a>
```

Comment:
Stop him to make don't make him lose so that I don't loose, hence I win

Reply:
we discussed it this morning, and clearly the feature is not very clear at best ^^